### PR TITLE
Add instructions for setting up the JDK

### PR DESCRIPTION
**Pull request by vinaysshenoy**
The `README` was missing setup instructions for the JDK. Added this in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,56 @@
 
 An Android app for recording blood pressure measurements.
 
+## Pre-requisites
+
+The application currently requires JDK 1.8 to build. If you already have JDK 1.8 installed, skip this step.
+
+**I am not sure if I have the JDK installed, or I am not sure if I have the right version installed**
```
Comment (sanchita) [inline README.md]: Can we add a title here which says `FAQs`?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,56 @@
...
+
+Run the command `java -version`. If you have the right version of the JDK installed, you should see something like:
+```sh
+openjdk version "1.8.0_232"
+OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_232-b09)
+OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.232-b09, mixed mode)
+```
+
+If this command has an error, or shows a different version, you can follow the instructions below to install the JDK.
```
Comment (sanchita) [inline README.md]: I think the answers to the questions do not need a margin on the left.
Wdyt?

File: README.md (hunk 1 above)
Comment (sanchita) [reply README.md]: This looks like a question. Should we add a `?` to all of them?
Or we can just rename them to 
1. `How to check JDK version?`
2. `How to install JDK?`

Comment (vinaysshenoy) [reply README.md]: Hmm, this is not exactly an FAQ, but sort of a POV of a reader who might be in a state of uncertainty about the JDK installation. I'll reword this to match your suggestions.

File: README.md (hunk 2 above)
Comment (vinaysshenoy) [reply README.md]: I have not added any margins, I think this is how it's getting rendered. Could you please check the raw file and confirm?

Comment (sanchita) [reply README.md]: Ok. No worries then.

File: README.md (hunk 1 above)
Comment (vinaysshenoy) [reply README.md]: Done.
